### PR TITLE
feat(types): add typings for `defaultAsComponent: true` case.

### PR DESCRIPTION
**Pull request by MrFoxPro**
https://github.com/jfgodoy/vite-plugin-solid-svg/issues/24

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,8 +36,12 @@ export default defineConfig({
 ```json
 // tsconfig.json
 "compilerOptions": {
-  // ...
-  "types": ["vite/client", "vite-plugin-solid-svg/types"],
+  "types": [
+    "vite/client",
+    "vite-plugin-solid-svg/types",
+    // Use vite-plugin-solid-svg/types-default together with `defaultAsComponent`
```
Comment (seahindeniz) [inline README.md:42]: Maybe `vite-plugin-solid-svg/as-component`?

Comment (MrFoxPro) [reply README.md:42]: Sounds nice, but for consistency will use `component-solid`.

**Comment by MrFoxPro** [comment]
@jfgodoy let's publish it

**Comment by jfgodoy** [comment]
great work!! :tada: I've  just published it.

**Pull request merged**